coredb-operator/tests/integration_tests.rs

```diff
@@ -0,0 +1,68 @@
+// Include the #[ignore] macro on slow tests.
+// That way, 'cargo test' does not run them by default.
+// To run just these tests, use 'cargo test -- --ignored'
+// To run all tests, use 'cargo test -- --include-ignored'
+//
+// https://doc.rust-lang.org/book/ch11-02-running-tests.html
+//
+// These tests assume there is already kubernetes running and you have a context configured.
+// It also assumes that the CRD(s) and operator are already installed for this cluster.
+// In this way, it can be used as a conformance test on a target, separate from installation.
+
+use kube::Api;
+use kube::api::{PatchParams, Patch};
+use controller::CoreDB;
+
+#[tokio::test]
+#[ignore]
+async fn functional_test_creates_running_pod() {
+    // Initialize the Kubernetes client
+    let client = kube_client().await;
+
+    let coredbs: Api<CoreDB> = Api::namespaced(client, "default");
+    let coredb_json = serde_json::json!({
+        "apiVersion": "kube.rs/v1",
+        "kind": "CoreDB",
+        "metadata": {
+            "name": "sample-coredb"
+        },
+        "spec": {
+            "replicas": 1
+        }
+    });
+    let params = PatchParams::apply("coredb-integration-test");
+    let patch = Patch::Apply(&coredb_json);
+    let coredb_resource = coredbs.patch("sample-coredb", &params, &patch).await;
+    panic!("{:?}", coredb_resource)
```

Comment:
got it working to create and panic here for debugging the resource. now looking into waiting - since need some logic like "check that a pod starts and becomes ready in the next 60 seconds" or something

Reply:
I was originally going down a too-deep path, I think this is the most abstract API available https://docs.rs/kube/latest/kube/runtime/wait/index.html

Reply:
I think I got that part working